packages/admin-api-client/README.md

```diff
@@ -0,0 +1,243 @@
+# @shopify/admin-api-client
+
+<!-- ![Build Status]() -->
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](../../LICENSE.md)
+
+<!-- [![npm version](https://badge.fury.io/js/%40shopify%2Fadmin-api-client.svg)](https://badge.fury.io/js/%40shopify%2Fadmin-api-client) -->
+
+The Admin API Client library is for developers who want to interact with Shopify's GraphQL `Admin API`. The features of this library are designed to be lightweight and minimally opinionated.
+
+## Getting Started
+
+To install this package, you can run this in your terminal:
+
+```typescript
+npm install @shopify/admin-api-client -s
+```
+
+## Admin API Client Examples
+
+### Initialize the Admin API Client
+
+```typescript
+import {createAdminAPIClient} from '@shopify/admin-api-client';
+
+const client = createAdminAPIClient({
```

Comment:
I think the README needs to be updated to use the newer type and function names (i.e. when we switched from `API` to `Api` for naming)